docs/modules/ROOT/pages/includes/quarkus-fury_quarkus.fury.adoc

```diff
@@ -41,5 +41,61 @@ endif::add-copy-button-to-env-var[]
 |boolean
 |`true`
 
+a|icon:lock[title=Fixed at build time] [[quarkus-fury_quarkus-fury-register-class-names]] [.property-path]##link:#quarkus-fury_quarkus-fury-register-class-names[`quarkus.fury.register-class-names`]##
+
+[.description]
+--
+Names of classes to register which no need to be with class-id or customize serializer.
+
+
+ifdef::add-copy-button-to-env-var[]
+Environment variable: env_var_with_copy_button:+++QUARKUS_FURY_REGISTER_CLASS_NAMES+++[]
+endif::add-copy-button-to-env-var[]
+ifndef::add-copy-button-to-env-var[]
+Environment variable: `+++QUARKUS_FURY_REGISTER_CLASS_NAMES+++`
+endif::add-copy-button-to-env-var[]
+--
+|string
+|
+
+h|[[quarkus-fury_section_quarkus-fury-register-classes]] [.section-name.section-level0]##link:#quarkus-fury_section_quarkus-fury-register-classes[Configurations of register classes]##
+h|Type
+h|Default
+
+a|icon:lock[title=Fixed at build time] [[quarkus-fury_quarkus-fury-register-classes-register-class-name-class-id]] [.property-path]##link:#quarkus-fury_quarkus-fury-register-classes-register-class-name-class-id[`quarkus.fury.register-classes."register-class-name".class-id`]##
+
+[.description]
+--
+Class id must be greater or equal to 256, and it must be different between classes. The default is -1.
+
+
+ifdef::add-copy-button-to-env-var[]
+Environment variable: env_var_with_copy_button:+++QUARKUS_FURY_REGISTER_CLASSES__REGISTER_CLASS_NAME__CLASS_ID+++[]
```

Comment:
```suggestion
Environment variable: env_var_with_copy_button:+++QUARKUS_FURY_REGISTER_CLASSES_REGISTER_CLASS_NAME_CLASS_ID+++[]
```

Reply:
No, This is generated by `quarkus-maven-plugin`. I think the double `_` means that the users need to replace it with the real class name. So the environment looks like `QUARKUS_FURY_REGISTER_CLASSES__io.quarkiverse.fury.it.Foo__CLASS_ID`.

@gastaldi is it right? I assume that this is similar with the logger configuration.

Reply:
Yes, @zhfeng is right